.github/actions_scripts/analyse_git_reference.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
```

Comment:
Python. Interesting. So you get some information from the current build/tag which triggered this?

Reply:
yes.
Easier string manipulations like startsWith.